### PR TITLE
Extended the configuration example in the README

### PR DESCRIPTION
**Pull request by lribi**
See: #78

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,10 +75,12 @@ exercise_html_purifier:
         custom:
             config:
                 Core.Encoding: 'ISO-8859-1'
+                HTML.Allowed: 'a[href|target],p,br'
+                Attr.AllowedFrameTargets: '_blank'
 ```
 
 In this example, a `exercise_html_purifier.custom` service will also be defined,
-which includes both the cache and encoding options. Available configuration
+which includes cache, encoding, HTML tags and attribute options. Available configuration
```
Comment (HeahDude) [inline README.md:83]:
```suggestion
which includes cache, encoding, HTML tags and attributes options. Available configuration
```

**Review by HeahDude — COMMENTED** [review]
Beside a minor comment this looks good to me.
Thank you for this contribution and happy holidays!

**Comment by lribi** [comment]
Thanks, you too.

**Comment by HeahDude** [comment]
Thanks @lribi

**Pull request merged**